app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -145,6 +145,9 @@ const CustomSelect: React.FC<ISelectProps> = ({
       }}
       MenuProps={{
         getContentAnchorEl: null,
+        style: {
+          zIndex: 1302,
```

Comment:
nit: can we do 999? feel free to ignore.

Reply:
for whatever reason our code uses 1301 in certain places, I think _because_ material ui uses this stuff.. I hesitate to push it any higher *AND* I would also recommend we don't push this into 6.7 because it there is a small chance it might cause a worse bug elsewhere.

Reply:
gotcha! looks good to me then 👍